app/src/main/java/com/nitramite/courier/DHLExpressStrategy.java

```diff
@@ -30,114 +33,146 @@ public class DHLExpressStrategy implements CourierStrategy {
     private static final String TAG = DHLExpressStrategy.class.getSimpleName();
 
 
+    private static final String cookieUrl = "https://www.dhl.com/M18S-b_-biKSNX9WPK72u-X7wuU/p1EN7cwhau/W2poWkgCAw/aQJv/OTZADXA";
```

Comment:
Somehow this URL looks like it's going to change often or is this just temporary?